app/views/blog/index.html.erb

```diff
@@ -1 +1,2 @@
 <h1><%= t('blog.page-title') %></h1>
+<%= render @posts %>
```

Comment:
Followed [rails guide example for rendering collections](https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections)